python/_fio_plugin/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/centos/centos:stream8
+
+RUN dnf -y module install python39 && dnf -y install python39 python39-pip git make gcc fio
```

Comment:
I don't think we need git, make, or gcc here, do we? Also, should we be explicit about the fio version, or is that getting too pedantic?

Reply:
I needed git for [pip requirements because arcaflow-plugin-sdk](https://github.com/mfleader/arcaflow-plugins/blob/967e265a8624941dc9bd79fba797c83d2bff7c82/python/_fio_plugin/requirements.txt#L1) is not currently a package on pypi. Was that not the case for you?

Reply:
I do not think specifying the fio version is too pedantic. I can add that

Reply:
how long is forever, truly?